src/main/resources/org/sagebionetworks/web/client/widget/team/JoinTeamConfigEditorViewImpl.ui.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui" 
+	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+	
+	<bh:Div>
+		<b:FieldSet>
+			<b:FormGroup>
+				<b:FormGroup>
+				    <b:FormLabel>Team</b:FormLabel>
+					<g:SimplePanel ui:field="suggestBoxPanel"/>
+				</b:FormGroup>
+				<b:FormGroup>
+				    <b:FormLabel for="joinButtonText">Wiki Text for Non-Members</b:FormLabel>
```

Comment:
Make capitalization consistent. "Wiki text for non-members" "\"Successfully joined\" notification wiki text", "\"Team request pending\" wiki text", ...